client/src/ui/molecules/breadcrumbs.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import { DocParent } from "../../document/types";
+
+import "./breadcrumbs.scss";
+
+export default function Breadcrumbs({ parents }: { parents: DocParent[] }) {
+  if (!parents.length) {
+    throw new Error("Empty parents array");
+  }
+
+  return (
+    <ol
+      typeof="BreadcrumbList"
+      vocab="https://schema.org/"
+      aria-label="breadcrumbs"
+      className="breadcrumbs"
+    >
+      {parents.map((parent, i) => {
+        let currentCrumb = i + 1;
```

Comment:
nit: Why `let` when everywhere else it's `const`?

Reply:
Doh! Forgot to change it to `const`